pkg/server/server_test.go

```diff
@@ -1563,3 +1563,60 @@ func TestDelegateCheckResolver(t *testing.T) {
 		require.True(t, ok)
 	})
 }
+
+func TestWriteAuthorizationModelWithExperimentalEnableModularModels(t *testing.T) {
+	ctx := context.Background()
```

Comment:
```suggestion
	t.Cleanup(func() {
		goleak.VerifyNone(t)
	})
	
	ctx := context.Background()
```
Please assert no leak.  Your changes should not cause leak - but by convention, please add the assertion to ensure no goroutine leak elsewhere

Reply:
Adding the call looks to catch a leak, however I added to another test (`invalid_schema_error_in_write_model`) that uses a mocked datastore in conjunction with `WriteAuthorizationModel` and it also errors there on main.

Given that other tests are using `WriteAuthorizationModel` with a non-mocked datastore is there any chance that could be the cause? If not, any tips how I can go about debugging this?

```
found unexpected goroutines:
        [Goroutine 130149 in state select, with github.com/karlseguin/ccache/v3.(*Cache[...]).worker on top of the stack:
        github.com/karlseguin/ccache/v3.(*Cache[...]).worker(0x103e960e0)
                /Users/ewan/go/pkg/mod/github.com/karlseguin/ccache/v3@v3.0.5/cache.go:243 +0x14c
        created by github.com/karlseguin/ccache/v3.New[...] in goroutine 130148
                /Users/ewan/go/pkg/mod/github.com/karlseguin/ccache/v3@v3.0.5/cache.go:65 +0x56c
         Goroutine 130151 in state select, with github.com/karlseguin/ccache/v3.(*Cache[...]).worker on top of the stack:
        github.com/karlseguin/ccache/v3.(*Cache[...]).worker(0x103e960e0)
                /Users/ewan/go/pkg/mod/github.com/karlseguin/ccache/v3@v3.0.5/cache.go:243 +0x14c
        created by github.com/karlseguin/ccache/v3.New[...] in goroutine 130150
                /Users/ewan/go/pkg/mod/github.com/karlseguin/ccache/v3@v3.0.5/cache.go:65 +0x56c
        ]
```

Reply:
Added in e7679cdab99fce275ceedfaf1a137458b08de86a